stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -351,62 +371,88 @@ Theses properties can be used to provide visual information or add interactions
+                  <Icon name="profile" invertColor />
+                </div>
+              }
+              lastElement={
+                <div style={{ display: 'grid', alignItems: 'center', height: '100%' }}>
+                  <Icon name="check" invertColor />
+                </div>
+              }
             />
             <ListItem
               title="Item 5"
               description="Elements can be button or simple text."
               firstElement={
-                <Button
-                  label="A button"
-                  backgroundColor="error"
-                  size="small"
-                  variant="icon"
-                  icon={<Icon name="cross" size="15" invertColor />}
-                  onClick={handleClickCross}
-                />
+                <div style={{ display: 'flex', alignItems: 'center', height: '100%' }}>
+                  <Button
+                    label="A button"
+                    backgroundColor="error"
+                    size="small"
+                    variant="icon"
+                    icon={<Icon name="cross" size="15" invertColor={isLightTheme} />}
+                    onClick={() => setOpen1(true)}
+                  />
+                </div>
               }
               lastElement={
-                <Typography as="div" color="inverted-text" fontSize="small" fontWeight="bold">
-                  This is a simple text
-                </Typography>
+                <div style={{ display: 'grid', alignItems: 'center', height: '100%' }}>
+                  <Typography as="div" color="inverted-text" fontSize="small" fontWeight="bold">
+                    This is a simple text
+                  </Typography>
+                </div>
               }
             />
             <ListItem
               title="Item 6"
               description="An other example with button"
-              firstElement={<Icon name="wallet" invertColor />}
+              firstElement={
+                <div style={{ display: 'grid', alignItems: 'center', height: '100%' }}>
+                  <Icon name="wallet" invertColor />
+                </div>
+              }
               lastElement={
-                <Button
-                  label="Select"
-                  backgroundColor="success"
-                  size="small"
-                  variant="secondary"
-                  onClick={handleClickSelect}
-                />
+                <div style={{ display: 'grid', alignItems: 'center', height: '100%' }}>
+                  <Button
+                    label="Select"
+                    backgroundColor="success"
+                    size="small"
+                    variant="secondary"
+                    onClick={() => setOpen2(true)}
+                  />
+                </div>
               }
             />
           </List>
           <Toast
             isOpened={open1}
-            onOpenChange={handleOpenChange1}
+            onOpenChange={() => setOpen1(false)}
```

Comment:
Could you remove lambdas from global jsx/mdx and turn them into functions?

Reply:
I was not aware this is unwanted. It is now fixed